prod/westeurope/notifications/functions_pushnotif/function_app/terragrunt.hcl

```diff
@@ -19,6 +19,15 @@ dependency "storage_notifications_queue_push-notifications" {
   config_path = "../../../internal/api/storage_notifications/queue_push-notifications"
 }
 
+# Test users storage
+dependency "storage_beta-test-users" {
```

Comment:
```suggestion
dependency "storage_beta_test_users" {
```

Reply:
@pasqualedevita  Do we need to change folder name as well?

Reply:
yes use `_` everywhere it's a terraform lint preference

Reply:
Done in [7976e4a](https://github.com/pagopa/io-infrastructure-live-new/pull/498/commits/7976e4a0ee82fa87febbab7aa83c61aa0549fa14)